src/index.html

```diff
@@ -11,7 +11,158 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>
+          Personal information
+        </legend>
+        <label class="rows">Surname:
+          <input
+           name="surname"
+           type="text"
+           autocomplete="off"
+           minlength="5"
+           >
```

Comment:
Close bracket `>` should have the same indentation with open bracket `<`
```suggestion
          <input
           name="surname"
           type="text"
           autocomplete="off"
           minlength="5"
          >
```